lib/filterx/expr-plus.c

```diff
@@ -37,9 +37,7 @@ _eval(FilterXExpr *s)
 
   FilterXObject *lhs_object = filterx_expr_eval(self->super.lhs);
```

Comment:
We should either make these `filterx_expr_eval_typed()` or support `message_value` in all the `add()` implementations.